.github/workflows/Test.yml

```diff
@@ -16,671 +16,692 @@ jobs:
+    steps:
+      - name: '${{ matrix.icon }} Setup MSYS2'
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW${{ matrix.bits }}
+          update: true
+          install: >-
+            tree
+            zstd
+            zip
+            tar
+
+      - name: '⚙ git config'
+        run: git config --global core.autocrlf input
+        shell: bash
+
+      - name: '🧰 Checkout'
+        uses: actions/checkout@v2
+
+      - name: '📥 Download artifact: package'
+        uses: actions/download-artifact@v2
+        with:
+          path: artifact
+          name: MINGW${{ matrix.bits }}-mcode
+
+      - name: '🛠 Install package'
+        run: pacman --noconfirm -U artifact/mingw-w64-${{ matrix.arch }}-ghdl-mcode-*.zst
+
+      - name: '🥡 Generate standalone zipfile'
+        run: |
+          _zipdir='MINGW${{ matrix.bits }}-mcode-standalone'
+          mkdir -p "${_zipdir}"-extract
+          tar xf artifact/mingw-w64-${{ matrix.arch }}-ghdl-mcode-*.zst -C "${_zipdir}"-extract
+          cd "${_zipdir}-extract/MINGW${{ matrix.bits }}"/bin
+          ../../../scripts/msys2-mcode/GetStandaloneDeps.sh
+          cd ../../..
+          mv "${_zipdir}"-extract/mingw${{ matrix.bits }} "${_zipdir}"
+          tree "${_zipdir}"
+          zip "${_zipdir}".zip -r "${_zipdir}"
+
+      - name: '📤 Upload artifact: zipfile'
+        uses: actions/upload-artifact@v2
+        with:
+          name: MINGW${{ matrix.bits }}-mcode-standalone
+          path: MINGW${{ matrix.bits }}-mcode-standalone.zip
 
 #
 # Windows CPython pyGHDL Test with MSYS2 installation
 #
 
   win-cpython-msys2:
-    needs: win-msys2-build-package
+    name: '🚦🐍${{ matrix.icon }} · ${{ matrix.pkg }}${{ matrix.bits }}'
     runs-on: windows-latest
+
+    needs:
+      - win-msys2-build-package
+
     strategy:
       fail-fast: false
       matrix:
-        include: [
-          #{icon: '🟪', pkg: 'llvm',  bits: '32', arch: i686 }, ! not yet functional
-          {icon: '🟦', pkg: 'llvm',  bits: '64', arch: x86_64, pyarch: x64 },
-          {icon: '🟪', pkg: 'mcode', bits: '32', arch: i686,   pyarch: x86 },
-          {icon: '🟦', pkg: 'mcode', bits: '64', arch: x86_64, pyarch: x64 }, #! simulation with mcode is not yet supported on win64
-        ]
-    name: '🚦🐍${{ matrix.icon }} · ${{ matrix.pkg }}${{ matrix.bits }}'
+        include:
+#          - {icon: '🟪', pkg: 'llvm',  bits: '32', arch: i686 }, ! not yet functional
+          - {icon: '🟦', pkg: 'llvm',  bits: '64', arch: x86_64, pyarch: x64 }
+          - {icon: '🟪', pkg: 'mcode', bits: '32', arch: i686,   pyarch: x86 }
+          - {icon: '🟦', pkg: 'mcode', bits: '64', arch: x86_64, pyarch: x64 } #! simulation with mcode is not yet supported on win64
+
     defaults:
       run:
         shell: pwsh
+
     steps:
+      - name: '${{ matrix.icon }} Setup MSYS2'
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW${{ matrix.bits }}
+          update: true
+
+      - name: '⚙ git config'
```

Comment:
This icon/emoji seems to be different from the other "gears". The same with the '🛠️' in the two steps below.

Reply:
Understood that the difference is because of the font used by the editor (web gui).